lib/sda/overlay.go

```diff
@@ -128,7 +128,7 @@ func (o *Overlay) StartNewNode(protocolID uuid.UUID, tree *Tree) (*Node, error)
 	// instantiate node
 	var err error
 	o.nodes[token.Id()], err = NewNode(o, token)
-	dbg.Lvl3("Making new node:", err)
+	dbg.Lvl3("Making new node (err=", err, ")")
 	if err != nil {
```

Comment:
Did not want to remove it since it was already there. But it is still quite useful to see a new node has been created .. ? As you like, I don't really care.